### PR TITLE
[Code Together] Adding SISWAP operation

### PR DESCRIPTION
**Pull request by charmerDark**
Hey @antalszava 
I've just added the SISWAP class, but I am having trouble exposing it as `qml.SISWAP` . I tried following the [PR for ISWAP](https://github.com/PennyLaneAI/pennylane/pull/1298/files). But I think the code structure has changed a bit since then? Could you please help me out on this end?

**Related issues**

Closes https://github.com/PennyLaneAI/pennylane/issues/1527

### CODE REVIEW

**Comment by antalszava** [comment]
Hi @charmerDark, after checking out your branch I could access `qml.SISWAP` :thinking: Was there a particular place in the codebase where it was not working?

The operations have indeed been split across files. If a new op is added to one of the existing files, it should be imported by in the top level `__init__.py` file.

**Comment by charmerDark** [comment]
Hey @antalszava !
I hadn't added any tests for SQISW because it redirects to SISWAP class. Also should I mention SQISW in the documentation line for SISWAP?

**Comment by antalszava** [comment]
Hi @charmerDark, just for extra safety, cases could be added for `qml.SQISW` too, similar to how it's the case for `CPhase`:
* https://github.com/PennyLaneAI/pennylane/blob/e44a3274a30b4ec6597b198703e306d25d8f7886/tests/ops/test_qubit_ops.py#L431
* https://github.com/PennyLaneAI/pennylane/blob/e44a3274a30b4ec6597b198703e306d25d8f7886/tests/ops/test_qubit_ops.py#L1556

It's mostly a matter of adding the same test case as for `qml.SISWAP`, just calling `qml.SQISW`.

> Also should I mention SQISW in the documentation line for SISWAP?

Yep, it could also be added to `doc/introduction/operations.rst`, just is the case with `CPhase`.

File: pennylane/ops/__init__.py [hunk 1]
```diff
@@ -79,6 +79,8 @@ def diagonalizing_gates(self):
         return []
 
 
+print(_qubit__all__)
+
```
Comment (antalszava) [inline pennylane/ops/__init__.py:83]:
```suggestion
```

**Comment by charmerDark** [comment]
Hey @antalszava
Could you see if its all right now? Thanks in advance

File: pennylane/ops/qubit/non_parametric_ops.py [hunk 2]
```diff
@@ -646,6 +646,69 @@ def adjoint(self):
         return ISWAP(wires=self.wires).inv()
 
 
+class SISWAP(Operation):
+    r"""SISWAP(wires)
+    The square root of i-swap operator. Can also be accessed as `qml.SQISW`
```
Comment (antalszava) [inline pennylane/ops/qubit/non_parametric_ops.py:651]:
```suggestion
    The square root of i-swap operator. Can also be accessed as ``qml.SQISW``
```

**Review by antalszava — CHANGES_REQUESTED** [review]
Hi @charmerDark, this is getting there! :tada: Left some suggestions, but the main addition is looking nice. :slightly_smiling_face:

File: .github/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 <h3>New features since last release</h3>
 
+* New SISWAP operation added to default_qubit device. Also added associated test [#1563](https://github.com/PennyLaneAI/pennylane/pull/1563)
```
Comment (antalszava) [inline .github/CHANGELOG.md:5]:
```suggestion
* Added a new `SISWAP` operation and a `SQISW` alias with support to the `default_qubit` device.
  [#1563](https://github.com/PennyLaneAI/pennylane/pull/1563)
```

**Review by antalszava — APPROVED** [review]
Hi @charmerDark, this is looking great! :tada: Thank you for the addition and for following through the review process. :blush: :1st_place_medal:

**Pull request merged**